test/libyul/objectCompiler/wasm/subObject.yul

```diff
@@ -0,0 +1,21 @@
+object "a" {
+  code {}
+  // Unreferenced data is not added to the assembled bytecode.
+  data "str" "Hello, World!"
+  object "sub" { code { function main() { i64.drop(11) } } }
+}
+// ====
+// wasm: true
+// ----
+// Text:
+// (module
+//     ;; custom section for sub-module
+//     ;; (@custom "sub" "286d6f64756c650a20202020286d656d6f727920246d656d6f727920286578706f727420226d656d6f727922292031290a20202020286578706f727420226d61696e22202866756e6320246d61696e29290a0a2866756e6320246d61696e0a2020202028626c6f636b20246c6162656c5f0a20202020202020202864726f7020286936342e636f6e737420313129290a20202020290a290a0a290a")
+//     ;; custom section for data
+//     ;; (@custom "str" "48656c6c6f2c20576f726c6421")
```

Comment:
This is a proposed syntax for encoding custom sections, but it was not ratified, hence kept it as a comment.